RELEASE.md

```diff
@@ -39,36 +39,16 @@ The result should look something like:
 Bump version
 ------------
 
-1. Determine a new version number, and define the `VERSION` variable.
+1. Change `TOPOLVM_VERSION` in `versions.mk` to the latest topolvm chart release tag. (e.g. topolvm-chart-vX.Y.Z)
```

Comment:
I'm sorry, but I overlooked that we still needed to update `TOPOLVM_VERSION` in my following comment.
https://github.com/topolvm/pvc-autoresizer/pull/236#pullrequestreview-1815025548

The previous procedure was easy to run because we were able to create a PR almost by copy and pasete. So, I would like you to revert these changes and just delete the step 3 in the previous procedure.

Reply:
refrected it